packages/app/src/studio/StudioPanel.vue

```diff
@@ -79,11 +82,11 @@ const maybeRenderReactComponent = () => {
     studioSessionId: props.cloudStudioSessionId,
   })
 
-  if (!reactRoot.value) {
-    reactRoot.value = window.UnifiedRunner.ReactDOM.createRoot(container.value)
+  if (!container.value._studioReactRoot) {
```

Comment:
```suggestion
  // Store the react root on the container. We do this so that we have a reference to it that's
  // tied to the container value but absolutely do not want to use vue to do the tracking.
  // If vue tracks it (e.g. using a ref) it creates proxies that do not play nicely with React in
  // production
  if (!container.value._studioReactRoot) {
```